pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -723,9 +716,10 @@ func testPrereleaseExtensionUpgrade(t *testing.T) {
 		}
 		t.Logf("successfully tested extension upgrade flow with --upgrade-prereleases-extensions true")
 	}()
+	os.Exit(0)
```

Comment:
why do we need these exits?

Reply:
While performing the tests locally to test flakiness. I added them for faster iteration. I noticed after logging the last success log message the exit of test func takes 20-30 secs. So adding it made the test execution faster. 

But removing them in the PR as we don't need them here.